sdk/storagesync/Azure.ResourceManager.StorageSync/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0(2025-05-01)
```

Comment:
The CI is failing due to [Azure.ResourceManager.StorageSync.csproj](https://github.com/Azure/azure-sdk-for-net/blob/238e4626ce8d6d3458d2dcbf73cc23f7b77364d1/sdk/storagesync/Azure.ResourceManager.StorageSync/src/Azure.ResourceManager.StorageSync.csproj#L3) version doesn't match with this entry